tutorials/rendering/jitter_stutter.rst

```diff
@@ -241,6 +459,127 @@ If your mouse offers multiple :abbr:`DPI (Dots Per Inch)` settings, consider als
 On Linux, disabling compositing in window managers that allow it (such as KWin
 or Xfwm) can reduce input lag significantly.
 
+Latency Reduction
+~~~~~~~~~~~~~~~~~
+
+Starting with Godot 4.5, new pacing methods were introduced to reduce latency (and alleviate Jitter):
+
+1. ``rendering/rendering_device/vsync/latency_mode``. It supports 4 options:
+
+  * ``low_extreme`` (only available through the GDScript API and command line interface. Cannot be set by default).
+  * ``low`` (default).
+  * ``medium``.
+  * ``high_throughput``.
+
+2. ``PacingMethod``, a fallback solution when Waitable Swapchains is not available:
+
+  * ``SEQUENTIAL``: The CPU always stalls for the GPU to minimize latency. **This can heavily penalize framerate and does not always result in lower latency**. This method can only reduce latency if the system was already fast enough to hit V-Sync's framerate.
+  * ``PARALLEL``: The CPU and GPU try to run in parallel. This is the usual method of 2D and 3D rendering.
+  * ``AUTO`` automatically selects between SEQUENTIAL or PARALLEL based on current CPU and GPU performance. AUTO will only select SEQUENTIAL if the system is fast enough.
+
+Waitable Swapchains are by far the superior method, but if that's not available AUTO/SEQUENTIAL will be used instead.
+
+.. warning::
+
+    In all cases (*including* Waitable Swapchain method), **lowering latency implies sacrificing framerate**.
+    The question is, how much FPS (frames per second) are you willing to sacrifice for latency. The relationship is not linear,
+    which means the lower the latency you want to achieve the greater the FPS sacrifice, which can get ridiculously high.
+    This is not a bug, it is just the nature of how it works.
+
+.. warning::
+
+    Lowering latency makes Godot more susceptible to **microstutter**.
+    Thus if you're experiencing microstutter, use these options to *increase* latency instead.
+
+.. warning::
+    ``low_extreme`` might get you better latency (or even be a placebo) but always at a very large FPS cost.
+    Furthermore it may reintroduce microstutter. Caution is advised when seeking to lower latency too much.
+
+.. warning::
+    Just because ``low_extreme`` works great in your machine doesn't mean it will work fine
+    in other machines. Don't deploy to end users with this setting as a default.
```

Comment:
Collapse the two warnings together:

```suggestion
.. warning::

    ``low_extreme`` might get you better latency (or even be a placebo) but always at a very large FPS cost.
    Furthermore, it may reintroduce microstutter. Caution is advised when seeking to lower latency too much.

    Just because ``low_extreme`` works great in your machine doesn't mean it will work fine
    in other machines. Don't deploy to end users with this setting as a default.
```

In general, we should avoid having multiple adominition blocks next to each other. Having 2 is sometimes acceptable, but we should definitely never have 3 or more.